### ISSUE TITLE
Compilation of hidapi on Raspberry Pi

### ISSUE THREAD

**Issue by GiantMolecularCloud** [vfilimonov/co2meter#5]
Note that hidapi is a cython package and cython is not installed by default on Raspbian Stretch Lite.
I'd suggest adding a note to README.md:
> Note that hidapi requires cython (`sudo apt-get install cython`) to be installed, otherwise `pip install hidapi` will result in an `error code 1` in gcc.

Such a short note would have saved me at least an hour of wondering why it worked on a macbook but not a Raspberry Pi 3 B+ and might help others as well.

**Comment by vfilimonov** [comment]
@GiantMolecularCloud , thank you for the reporting! I'll add a note to readme that this problem might occur.

It is quite weird though - hidapi's setup.py has cython as a pre-requisite: https://github.com/trezor/cython-hidapi/blob/master/setup.py, so it should be installed by pip automatically...

**Comment by GiantMolecularCloud** [comment]
I've tested this a bit further by re-installing Raspbian and co2meter. Likely, the problem wasn't cython but libudev-dev. While testing both of them were installed as dependencies and co2meter worked afterwards but I only noticed cython to get installed.

Anyway, if other people have similar problems: the following installation routine works on a fresh Raspbian Stretch Lite
```sudo apt-get update
sudo apt-get upgrade
sudo apt-get install python-dev cython libudev-dev libusb-1.0-0 libusb-dev libusb-1.0-0-dev libhidapi-dev libavahi-compat-libdnssd-dev
sudo pip install cython hidapi co2meter flask pandas HAP-python
```
